sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1210,15 +1210,15 @@ def test_non_deterministic_estimator_skip_tests():
     # check estimators with non_deterministic tag set to True
     # will skip certain tests, refer to issue #22313 for details
     for est in [MinimalTransformer, MinimalRegressor, MinimalClassifier]:
-        all_tests = list(_yield_all_checks(est()))
+        all_tests = list(_yield_all_checks(est(), legacy=True))
```

Comment:
Why do we set explicitely the flag here?

Reply:
I wanted to be explicit to make sure future behavior changes don't affect this one, in case we disable legacy as a default.